lib/taiko.js

```diff
@@ -3338,9 +3338,17 @@ const evaluate = async (selector, func) => {
 };
 
 const validate = () => {
-  if (!dom || !page) {
+  if (!_client) {
     throw new Error('Browser or page not initialized. Call `openBrowser()` before using this API');
   }
+  if (_client._ws.readyState > 1) {
+    if (chromeProcess && chromeProcess.killed) {
+      throw new Error(
+        'Browser is closed due to `closeBrowser()` or crash. Try reopening by `openBrowser()`',
```

Comment:
Suggestion:

`The Browser instance was closed either via 'closeBrowser()' call, or it crashed for reasons unknown to Taiko. You can try launching a fresh instance using 'openBrowser()' or inspect the logs for details of the possible crash.`